packages/cgb-scripts/template/src/blocks.front-only.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Gutenberg Blocks
+ *
+ * All frontend blocks related JavaScript files should be imported here.
+ * These scripts will be enqueued on the frontend AND in the editor.
```

Comment:
It says `front-only` and then in docs you say ` frontend AND in the editor`. That confuses me. 🤔